server/controllers/cancellationsController.test.ts

```diff
@@ -172,7 +172,7 @@ describe('cancellationsController', () => {
       expect(cancellationService.getCancellation).toHaveBeenCalledWith(token, premisesId, bookingId, cancellation.id)
       expect(bookingService.getBooking).toHaveBeenCalledWith(token, premisesId, bookingId)
 
-      expect(response.render).toHaveBeenCalledWith('cancellations/confirm', { cancellation, booking })
+      expect(response.render).toHaveBeenCalledWith('cancellations/confirm', { cancellation, booking, premisesId })
```

Comment:
it definitely makes sense that a Booking object has a premisesId but I wonder if the upheaval it would cause outweighs the benefits it would give us as this point in time. I believe it would require backend changes as well?

Reply:
Yeah, it's totally not work it now 😆